src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -204,6 +206,8 @@ components:
       required:
         - stepReference
         - title
+        - targetDate
+        - targetDateRange
```

Comment:
Both fields will be populated in the response.

Reply:
When you say "both fields will be populated in the response" .... it looks like you've removed `targetDate` from the response?

Reply:
Ignore that, the comment's out of date, I should have deleted it.

Reply:
Ah, got'cha 👍